File: rmt/src/test/scala/de/flower/rmt/ui/player/page/events/EventPageTest.scala
package de.flower.rmt.ui.player.page.events

import de.flower.rmt.test.WicketTests
import org.testng.annotations.Test
import de.flower.rmt.ui.model.EventModel

/**
 * 
 * @author flowerrrr
 */

class EventPageTest extends WicketTests {

    @Test
    def renderPage() {
        val event = testData.createEvent()
        wicketTester.startPage(new EventPage(new EventModel(event)))
        wicketTester.dumpPage()
    }


}

File: rmt/src/test/scala/de/flower/rmt/ui/player/page/event/EventPageTest.java
package de.flower.rmt.ui.player.page.event;

import de.flower.rmt.model.event.Event;
import de.flower.rmt.test.AbstractWicketTests;
import de.flower.rmt.ui.model.EventModel;
import org.testng.annotations.Test;

/**
 * @author flowerrrr
 */

public class EventPageTest extends AbstractWicketTests {

    @Test
    public void testRender() {
        Event event = testData.createEventWithResponses();
        wicketTester.startPage(new EventPage(new EventModel(event)));
        wicketTester.dumpPage();
    }
}